prestoadmin/main.py

```diff
@@ -565,6 +575,10 @@ def run_tasks(task_list):
                                  % name)
                 display_command(name, 2)
 
+            if state.env.rpm_source and name.strip() != 'server.install':
```

Comment:
I'd structure this the same as the above (nodeps) for consistency even though it's a list of one at the moment.